ipa-core/src/net/transport.rs

```diff
@@ -152,30 +187,88 @@ impl HttpTransport {
             r.await
         }
     }
+}
+
+impl MpcHttpTransport {
+    #[must_use]
+    pub fn new(
+        http_runtime: IpaRuntime,
+        identity: HelperIdentity,
+        server_config: ServerConfig,
+        network_config: NetworkConfig<Helper>,
+        clients: [MpcHelperClient; 3],
+        handler: Option<HandlerRef<HelperIdentity>>,
+    ) -> (Self, MpcHelperServer<Helper>) {
+        let transport = Self::new_internal(http_runtime, identity, clients, handler);
+        let server = MpcHelperServer::new_mpc(&transport, server_config, network_config);
+        (transport, server)
+    }
+
+    fn new_internal(
```

Comment:
shouldn't we inline it?

Reply:
I would just leave that to the compiler...  This code-path is hardly on the hot path.

Reply:
it is not a performance concern, but a question about readability. It seems that `new_internal` is not used anywhere except `new` and it is not doing anything sophisticated

Reply:
Ahh ok. This is the way this was organized before but I agree it makes little sense to keep it separate.